src/components/Buttons/Button.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import Button from "./Button";
+
+export default {
+  component: Button,
+  title: "Components/Button",
+};
+
+export const Normal = () => <Button size="normal" text="Filter" />;
+export const Wide = () => <Button size="wide" text="hallo" />;
```

Comment:
This is the only button starting with a lower case letter. Here it doesn't matter, but might be important for later.